cmd/permissions-checker/permissionschecks.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os/user"
+	"regexp"
+	"strings"
+	"syscall"
+)
+
+// FindRecursive iterate in a directory (path) showing permissions in a recursive way
+/* path  - where to look for files and directories
+ * level - used to print the output in a hierarchical way
+ */
+func FindRecursive(path, level string) {
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// In verbose mode we are printing the output in a hierarchical way
+	if Verbose {
```

Comment:
I would increase the level always, not only when the verbose flag is enabled.
Also, given that this only uses spaces I would replace the string with an integer.
You can write a small helper that formats the message and the level in the way you want.
You may also find https://golang.org/pkg/strings/#Repeat useful

Reply:
Now I am using an `int` and the `repeat` function to convert it into space, about increasing the level only when verbose is enable, I am doing this because if verbose is enable I am printing the output in a hierarchical way, but if not, I am printing the output at the same level, so if I increase the level always, I need to implement a logic in the print function to differentiate when to use the level and when not.